src/Application/Common/Interfaces/INomadService.cs

```diff
@@ -0,0 +1,9 @@
+using Hippo.Application.Jobs;
+
+namespace Hippo.Application.Common.Interfaces;
+
+public interface INomadService
```

Comment:
Should this be named with Nomad in the name or not? Maybe it should be IJobService, Same for bindle service, maybe should be storage service?

Reply:
Will change this on another branch after this has been merged, as discussed